.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "files.exclude": {
+    "**/node_modules": false,
```

Comment:
In exercise 10, it took me a bit to figure out that `node_modules/str-utils/index.js` referred to `exercises/exercise-10/node_modules`, not the top-level `node_modules`. Adding this to my vscode settings made it easier to find.

Reply:
Would you be able to make a pull-request with this change?